index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
 </p>
 
-<p><dfn class="lint-ignore export">LTR</dfn> stands for "left-to-right" and refers to the inline base direction of left-to-right [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the left side of the page in horizontal text. It's used for scripts such as Latin, Cyrillic, Devanagari, and many others.</p>
+<p><dfn class="lint-ignore export">LTR</dfn>. Stands for "left-to-right" and refers to the inline base direction of left-to-right [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the left side of the page in horizontal text. It's used for scripts such as Latin, Cyrillic, Devanagari, and many others.</p>
 
 
 
 
 <div class="letter_anchor" id="m">M</div>
 
-<p><dfn data-lt="metadata|metadata" id="def_metadata" class="lint-ignore export">Metadata</dfn> is additional information about data. Key types of metadata for <a class="termref" href="#internationalization">internationalization</a> are <a class="termref" href="#def_language_metadata">language metadata</a> and metadata to support <a class="termref" href="#def_bidirectional_text">bidirectional text</a>. Metadata has a scope, e.g., a string or a set of strings. In absence of explicit metadata, defaults might apply, e.g. defaults for the <a class="termref" href="#def_base_direction">base direction</a> of a text.</p>
+<p><dfn class="lint-ignore export">Metadata</dfn>. Additional information about data. Key types of metadata for [=internationalization=] are [=language metadata=] and metadata to support [=bidirectional text=]. Metadata has a scope, e.g., a string or a set of strings. In absence of explicit metadata, defaults might apply, e.g. defaults for the [=base direction=] of a text.</p>
+
+<p><dfn class="lint-ignore export">Mojibake</dfn> (<span lang="ja">文字化け</span>). Garbled or incorrectly rendered or processed characters, generally caused by using the wrong <a>character encoding</a> to interpret the bytes in a string or file. The word is Japanese in origin and is pronounced <q>/mo.d&#x0361;&#x0292;i.ba.ke/</q>. For example, the word <q><span lang="ja">&#x6587;&#x5B57;&#x5316;&#x3051;</span></q> encoded as UTF-8 might be displayed as <q>&#x00e6;&#x0096;&#x0087;&#x00e5;&#x00ad;&#x0097;&#x00e5;&#x008c;&#x0096;&#x00e3;&#x0091;</q> if viewed in an application that thinks (incorrectly) that the character encoding is <code>ISO-8859-1</code>.</p>
+
+
+
+
+<div class="letter_anchor" id="o">O</div>
+	
+<p><dfn class="lint-ignore export">Othographic syllable</dfn> This term is not clearly defined in the Unicode Standard. For their materials, the i18n group defines it to mean a typographic unit that includes more than one [=grapheme cluster=]. This is commonly the case for Brahmi-derived scripts, such as Devanagari, or Balinese, when they form conjuncts or stacks.</p>
```

Comment:
Two general comments:

i. I would put the definition first and then explain the scope of it after.
ii. I found the "their materials" and "the i18n group" part unclear.

```suggestion
<p><dfn class="lint-ignore export">Othographic syllable</dfn> A [=typographic unit=] that includes more than one [=grapheme cluster=]. These are most commonly found in Brahmi-derived scripts, such as Devanagari, or Balinese, when forming conjuncts or stacks. This term is not defined in the Unicode Standard: it is defined here for use by W3C documents, such as language requirements.</p>
```

Reply:
Done.